metadata-ingestion/setup.py

```diff
@@ -471,6 +477,7 @@
     "fivetran": snowflake_common | bigquery_common | sqlglot_lib,
     "qlik-sense": sqlglot_lib | {"requests", "websocket-client"},
     "sigma": sqlglot_lib | {"requests"},
+    "sac": sac,
```

Comment:
imo sac isn't a sufficiently well-know abbreviation - let's name the source `sap-analytics-cloud` or `sap-sac`

Reply:
At least in context of SAP, SAC ist a common abbreviation for SAP Analytics Cloud...if I search for SAC in Google or Bing in an incognito window, besides the Swiss Alpine Club (😂), the next results are for the SAP Analytics Cloud (but of course there is still some kind of geo localization and the results are probably specific for Germany).

-> sap-sac is definitely wrong (the S in SAC stands for SAP) and sap-analytics-cloud is too long...

As the abbreviation would be also used for the platform urn, I would like to stick with SAC...;-)